src/components/BasicHead.vue

```diff
@@ -244,6 +241,20 @@ export default {
   },
   
   methods: {
+    defaults () {
+      let features = ['print', 'bookmark', 'collect']
+      this.featuresMenu = []
+      for(let feature of features) {
+        if(this.tool[feature]) {
+          this.featuresMenu.push(Object.assign({}, 
+            this.tool[feature], 
+            {title: feature, click: this[this.tool[feature].click]}))
+        }
+      }
+      if(this.tool.select.active) {
+        this.select = this.tool.select.initial
+      }
+    },
     printMap () {
```

Comment:
we may not be printing a map - just call this `print`

Reply:
Done